browser/resources/settings/brave_overrides/privacy_page.js

```diff
@@ -36,6 +36,17 @@ RegisterPolymerTemplateModifications({
           </settings-subpage>
         </template>
       `)
+      const isDeAmpingEnabled = loadTimeData.getBoolean('isDeAmpingEnabled')
```

Comment:
nit: this should be `isDeAmpFeatureEnabled` both for consistency (isIdleDetectionFeatureEnabled, etc..) and also to avoid confusing with the enabled state based on the pref